core/controllers/contributor_dashboard.py

```diff
@@ -310,107 +310,91 @@ class ReviewableOpportunitiesHandler(
     def get(self) -> None:
         """Fetches reviewable translation suggestions."""
         assert self.normalized_request is not None
-        topic_name = self.normalized_request.get('topic_name')
+        # Default value is None, since this is a GET request handler, which
+        # means all request parameters come in as strings.
+        topic_name = self.normalized_request.get('topic_name', None)
         language = self.normalized_request.get('language_code')
         opportunity_dicts: List[
             opportunity_domain.PartialExplorationOpportunitySummaryDict
             ] = []
         if self.user_id:
             for opp in (
                 self._get_reviewable_exploration_opportunity_summaries(
-                self.user_id, topic_name, language
-            )):
-                if opp is not None:
-                    opportunity_dicts.append(opp.to_dict())
+                self.user_id, topic_name, language)
```

Comment:
This should probably be indented by 4 from the previous line since it's a new scope. (Or extract the result of the 'get' call into a separate variable.)

Reply:
@StephenYu2018 Did you push your changes? There are a bunch of comments that say "Done" but I don't think it's done.